R/bambu.R

```diff
@@ -85,9 +85,16 @@
 bambu <- function(reads = NULL, rcFile = NULL, rcOutDir = NULL,
     annotations = NULL, genome = NULL, stranded = FALSE, ncore = 1,
     yieldSize = NULL, opt.discovery = NULL, opt.em = NULL,
-    discovery = TRUE, verbose = FALSE) {
-    annotations <- checkInputs(annotations, reads, readClass.file = rcFile, 
+    discoveryOnly = FALSE, quantOnly = FALSE, verbose = FALSE) {
+    if (discoveryOnly & quantOnly) {
+        warning(paste0("At least 1 of discoveryOnly and quantOnly must be FALSE. 
+ Rerun with either 1 or both parameters as FALSE"))
+        break
```

Comment:
stop("At least 1 of discoveryOnly and quantOnly must be FALSE. 
 Rerun with either 1 or both parameters as FALSE")